docs/tutorials/hello-world-application/creating-the-database.mdx

```diff
@@ -27,27 +38,53 @@ In the Kuneiform IDE, you must connect to the endpoint for the Kwil Network you
 
 Next, add the network details in the popup Menu. After you click "OK", the IDE will automatically switch to the new network.
 
+> **TODO** Believe that https://provider.kwil.com would make sense for a 1st pass Tutorial example as it does NOT require the user to set up their own network as a prerequisite.
```

Comment:
I agree that provider.kwil.com is best for a first pass tutorial; however, for these docs in pre-release, we want to highlight HTTP://localhost:8080 because a local kwild node is not compatible with what is currently deployed on https://provider.kwil.com. So if someone is trying to run this tutorial with everything else that is in these docs, they need to run a local kwil node.

Reply:
Once we do the next public release, this tutorial can go back to https://provider.kwil.com; however, because we have partners that are using the pre-release, everything in these docs should be consistent.